bedrock/settings/__init__.py

```diff
@@ -6,8 +6,8 @@
 
 from .base import *  # noqa
 
-if os.getenv('TRAVIS', False):
-    from .travis import *  # noqa
+if os.getenv('CIRCLECI', False):
+    from .circleci import *  # noqa
```

Comment:
I'd prefer to eliminate circleci.py and use [environment variables](https://circleci.com/docs/environment-variables).